packages/editor/src/components/sidebar/index.js

```diff
@@ -167,6 +119,10 @@ const SidebarContent = ( {
 					/>
 					{ ! isEditingTemplate && <PostSummary /> }
 					<PluginDocumentSettingPanel.Slot />
+					{ renderingMode !== 'post-only' && (
+						<TemplateContentPanel />
+					) }
+					<PostTransformPanel />
```

Comment:
Both of these panels were site editor specific and I'm guessing they can show up in the post editor as well now.